server/dfr3-service/src/main/java/edu/illinois/ncsa/incore/service/dfr3/models/DFR3Set.java

```diff
@@ -31,6 +31,7 @@ public abstract class DFR3Set {
 
     protected String hazardType;
     protected String inventoryType;
+    protected String dataType;
```

Comment:
Is this just to indicate the minimum dataType to use this fragility or is it the only one that should work with this fragility?